### DISCUSSION TITLE
Mobile browser does not load modal

### DISCUSSION THREAD

**Discussion by iRyteRong** [Web3Auth/Web3Auth#850] [Help]
I have tried several times to make this work. None of the examples work on mobile browsers. On Android, the modal comes up and just spins. On iOS browsers, there is an Error "Can't find variable BigInt". I used both Nextjs and react examples. Everything seems to work on the desktops.

**Comment by rinzler99** [comment]
Are you using the latest web3auth sdk because we have released a fix for bigint already.

**Comment by himanshuchawla009** [comment]
Which ios browser are you using , i believe you are using a bit older browsers. Web3auth works best with safari browsers > ios 13.
